enterprise/tailnet/pgcoord.go

```diff
@@ -1494,7 +1494,7 @@ func (h *heartbeats) sendBeats() {
 
 func (h *heartbeats) sendBeat() {
 	_, err := h.store.UpsertTailnetCoordinator(h.ctx, h.self)
-	if xerrors.Is(err, context.Canceled) {
+	if xerrors.Is(err, context.Canceled) || database.IsQueryCanceledError(err) {
```

Comment:
context.Canceled is included in `IsQueryCanceledError()`

Reply:
Even better!